internal/controller/operator/factory/vmanomaly/models.go

```diff
@@ -0,0 +1,247 @@
+package vmanomaly
+
+import (
+	"time"
+)
+
+type modelDetectionDirection string
+
+const (
+	bothModelDirection          modelDetectionDirection = "both"           //nolint:unused
+	aboveExpectedModelDirection modelDetectionDirection = "above_expected" //nolint:unused
+	belowExpectedModelDirection modelDetectionDirection = "below_expected" //nolint:unused
+)
+
+type CommonModelParams struct {
```

Comment:
Should it be private struct?

Reply:
i'd like to make it private, but `json:",inline"` requires embedded struct